tests/smoke/support/page/files/allFiles.ts

```diff
@@ -187,4 +213,132 @@ export class AllFilesPage {
     })
     await page.goto(startUrl)
   }
+
+  async resourceExist({ name }: { name: string }): Promise<boolean> {
```

Comment:
those things are not page specific (i hope i'm not wrong) ;) we already have a cta to navigate to an folder and a helper to check if a resource is there. Why not use them combined in the gherkin step?

Reply:
The result is a universal method, which, yes probably not in the right place. Calling methods from `cta` in `step definition` didn't work out nicely for me (I need your help). 
add the Actor interface to app.files?

Reply:
ok to get this moving i would say keep it there and refactor it in the same breath as numberOfVersions